### DISCUSSION TITLE
Exposure Controller acting oddly

### DISCUSSION THREAD

**Discussion by bonnef** [rsjaffe/MIDI2LR#776] [Q&A]
Hello all! This is my first time posting here; I hope this is the correct place.

I have been using MIDI2LR on my Behringer mini for years, and this problem seems to be getting progressively worse.

About half of the time, when I first use an adjustment knob on an image (exposure, white balance, etc.), it snaps the setting back to the minimum allowed instead of starting with where the setting currently is. Also, often, the values controlled by the knobs jump suddenly down instead of up and just in general don't move with much accuracy.

For reference, I have tried both pickup on and off. I did just do a hard reset on the Behringer, and that didn't seem to change anything. After the latest update, I was getting a "exposure controller not synchronized" error when trying to use that knob, but something I fiddled with seems to have fixed that for now.
I also just loaded the most recent version of Joseph Naim's preset (layers a and b) onto my Behringer, and I'm still having the issues.

Has anyone run into similar issues and found a fix?

One more question: while looking in the x-touch mini editor software I noticed the knob settings set to "pan". Is that the best setting for these? I found some people online using single for these. 

Thank you!

**Comment by antzpantz** [comment]
There was a new release earlier today that may be related to your issue.
Can you try that one and see if it fixes your problem?

**Reply by bonnef** [reply]
6.0.2.0? I do have that installed.

**Reply by RBCAMERA** [reply]
I also have the new release applied and also have observed that the jumping back slider problem still persists.  I believe it developed after a LR update a few months ago.  So frustrating because otherwise Midi2LR has been fantastic..   I use an X-touch mini and a Mac Mini M2 pro..   always the latest version of LR.. The jumping slider problem seems worse on Exposure and Contrast and then less so on Highlights.      Anyone have an idea.. ( I did trty a factory reset of my X-Touch Mini.. no change.)
